onebusaway-android/src/main/java/org/onebusaway/android/util/UIUtils.java

```diff
@@ -1141,4 +1138,19 @@ public static String formatTime(Context context, long time) {
                         DateUtils.FORMAT_NO_MIDNIGHT
         );
     }
+
+    /**
+     * Set smaller text size if the route short name has more than 3 characters
+     *
+     * @param view Text view
+     * @param routeShortName Route short name
+     */
+    public static void setTextSizeForArrivalsListAdapterStyleARoute(TextView view,
```

Comment:
Actually, to avoid confusion with `routeLongName`, let's call the method `maybeShrinkRouteName()`.